modules/navier_stokes/test/tests/finite_volume/ins/block_restriction/2d-rc.i

```diff
@@ -38,8 +38,8 @@ restricted_blocks = '1'
 []
 
 [Problem]
-  kernel_coverage_check = false
   fv_bcs_integrity_check = true
+  kernel_coverage_check = false
```

Comment:
So this one is needed?

Reply:
yes because block 2 ends up with no variables so it hits the error.
I could also just add a second non-block restricted variable

Reply:
Nah that's fine